vonk/features/terminologyintegration.rst

```diff
@@ -0,0 +1,100 @@
+.. _feature_terminologyintegration:
+
+External terminology services
+=============================
+
+.. warning:: The TerminologyIntegration plugin does not work together with the Vonk Terminology plugin. 
+
+To be able to use more elaborate terminology support, 
+we provide the option to integrate external FHIR terminology services with Vonk. 
+
+Vonk can redirect the following terminology operations to external services:
+* $validate-code
+* $expand
+* $lookup
+* $translate
+* $subsumes
+
+To enable the TerminologyIntegration it needs to be added to the PipelineOptions in the appsettings.
+
+.. note::
+Adding Vonk.Plugins.TerminologyIntegration means you cannot use Vonk.Plugins.Terminology. Vonk will log a warning if you have both plugins included in the appsettings. 
```

Comment:
Indent this line to show it as note, and put the note below the appsettings example.